api/room/schema_query.py

```diff
@@ -405,15 +405,17 @@ def resolve_bookings_analytics_count(
             self, query, start_date, end_date, room_id=room_id)
         return analytics
 
+    @Auth.user_roles('Admin', 'Default User')
     def resolve_analytics_for_daily_room_events(
```

Comment:
Nice work on the PR @daniellamarr
I think you should use `**kwargs` to represent how arguments are passed to the function.

Reply:
@jesseinit I'm not sure I understand what you mean.

Reply:
The function takes quite a number of arguments, you could use `**kwargs` to hold these arguments instead of passing them individually

Reply:
I have made the changes